### DISCUSSION TITLE
Two sidebars in Gollum Wiki

### DISCUSSION THREAD

**Discussion by vrindanayak** [gollum/gollum#2074] [Q&A]
Hello,
We are using Gollum Wiki as a private repository for our company products' documentation. Right now we make use of a _Sidebar.md file in the root location with content in it as
```
[[TOC]]
```
which shows the table of contents with menu and sub menus for the sections on the current page and is placed on the right by default.

Is there a possibility to also add a second Sidebar to the left containing the directory structure (menus and sub menus) for entire product documentation (including several pages)

I searched in the list of issues / discussions, I did not find any references so far which can achieve this. Can someone please advise or point me in the right direction?

**Comment by benjaminwil** [comment]
Hey there! At this time there is only one sidebar that can be added. Sorry to disappoint you. 

My only idea would be to consider using the header or footer to present the directory structure somehow. But I think you would need to be creative to make it take up a small amount of space.
